configs/trip_mode_choice.csv

```diff
@@ -1,405 +1,405 @@
-Walk-express penalty for intermediate stops,@c_walk_express_penalty * ~(df.first_trip | df.first_trip),,,,,,,,,,,1,,,,,,,,,,
+Description,Expression,DRIVEALONEFREE,DRIVEALONEPAY,SHARED2FREE,SHARED2PAY,SHARED3FREE,SHARED3PAY,WALK,BIKE,WALK_LOC,WALK_LRF,WALK_EXP,WALK_HVY,WALK_COM,DRIVE_LOC,DRIVE_LRF,DRIVE_EXP,DRIVE_HVY,DRIVE_COM,TAXI,TNC_SINGLE,TNC_SHARED
+#Drive alone no toll,,,,,,,,,,,,,,,,,,,,,,
+DRIVEALONEFREE - Unavailable,sov_available == False,-999,,,,,,,,,,,,,,,,,,,,
+DRIVEALONEFREE - Unavailable for zero auto households,auto_ownership == 0,-999,,,,,,,,,,,,,,,,,,,,
+DRIVEALONEFREE - Unavailable for persons less than 16,age < 16,-999,,,,,,,,,,,,,,,,,,,,
+DRIVEALONEFREE - Unavailable for joint tours,is_joint == True,-999,,,,,,,,,,,,,,,,,,,,
+DRIVEALONEFREE - Unavailable if didn't drive to work,is_atwork_subtour & ~work_tour_is_SOV,-999,,,,,,,,,,,,,,,,,,,,
+DRIVEALONEFREE - In-vehicle time,@c_ivt*odt_skims['SOV_TIME'],1,,,,,,,,,,,,,,,,,,,,
+DRIVEALONEFREE - Terminal time,@c_walktimeshort * df.total_terminal_time,1,,,,,,,,,,,,,,,,,,,,
+DRIVEALONEFREE - Operating cost,@df.c_cost * costPerMile * odt_skims['SOV_DIST'],1,,,,,,,,,,,,,,,,,,,,
+DRIVEALONEFREE - Parking cost,c_cost * total_parking_cost,1,,,,,,,,,,,,,,,,,,,,
+DRIVEALONEFREE - Bridge toll,@df.c_cost * odt_skims['SOV_BTOLL'],1,,,,,,,,,,,,,,,,,,,,
+DRIVEALONEFREE - Person is between 16 and 19 years old,@c_age1619_da * ((df.age >= 16) & (df.age <= 19)),1,,,,,,,,,,,,,,,,,,,,
+#Drive alone toll,,,,,,,,,,,,,,,,,,,,,,
+DRIVEALONEPAY - Unavailable,sovtoll_available == False,,-999,,,,,,,,,,,,,,,,,,,
+DRIVEALONEPAY - Unavailable for zero auto households,auto_ownership == 0,,-999,,,,,,,,,,,,,,,,,,,
+DRIVEALONEPAY - Unavailable for persons less than 16,age < 16,,-999,,,,,,,,,,,,,,,,,,,
+DRIVEALONEPAY - Unavailable for joint tours,is_joint == True,,-999,,,,,,,,,,,,,,,,,,,
+DRIVEALONEPAY - Unavailable if didn't drive to work,is_atwork_subtour & ~work_tour_is_SOV,,-999,,,,,,,,,,,,,,,,,,,
+DRIVEALONEPAY - In-vehicle time,@c_ivt*odt_skims['SOVTOLL_TIME'],,1,,,,,,,,,,,,,,,,,,,
+DRIVEALONEPAY - Terminal time,@c_walktimeshort * df.total_terminal_time,,1,,,,,,,,,,,,,,,,,,,
+DRIVEALONEPAY - Operating cost,@df.c_cost * costPerMile * odt_skims['SOVTOLL_DIST'],,1,,,,,,,,,,,,,,,,,,,
+DRIVEALONEPAY - Parking cost,c_cost * total_parking_cost,,1,,,,,,,,,,,,,,,,,,,
+DRIVEALONEPAY - Bridge toll,@df.c_cost * odt_skims['SOVTOLL_BTOLL'],,1,,,,,,,,,,,,,,,,,,,
+DRIVEALONEPAY - Value toll,@df.c_cost * odt_skims['SOVTOLL_VTOLL'] + cordon_toll  * df.is_dest_cordon - income_low_100_discount * df.is_income_lower_100,,1,,,,,,,,,,,,,,,,,,,
+DRIVEALONEPAY - Person is between 16 and 19 years old,@c_age1619_da * ((df.age >= 16) & (df.age <= 19)),,1,,,,,,,,,,,,,,,,,,,
+#Shared ride 2,,,,,,,,,,,,,,,,,,,,,,
+SHARED2FREE - Unavailable,hov2_available == False,,,-999,,,,,,,,,,,,,,,,,,

```

Comment:
Could also add a new row here marking SHARED2FREE as unavailable for trips to the cordon area

Reply:
That's right, done !